ocaml/typing/typecore.ml

```diff
@@ -7583,7 +7583,9 @@ and type_apply_arg env ~app_loc ~funct ~index ~position_and_mode ~partial_app (l
           (lbl, Arg (arg, Mode.Value.legacy, sort_arg))
       | Position _ ->
           let arg = src_pos (Location.ghostify app_loc) [] env in
-          (* CR src_pos: Confirm that global value mode is correct *)
+          (* XXX jrodri for goldfirere: I think using [Mode.Value.legacy] here is correct,
```

Comment:
Yes, it's correct, by analogy with optional arguments.

Reply:
Cool sounds good. I have removed the CR question. Thanks!